package.json

```diff
@@ -103,6 +103,18 @@
       "browser": "./lib/search.js",
       "default": "./src/search.tsx"
     },
+    "#search-form": {
+      "browser": "./lib/search-form.js",
+      "default": "./src/search-form.tsx"
+    },
```

Comment:
You can actually use wildcards for these:

```
"#*": {
  "browser": "./lib/*.js",
  "default": "./src/*.tsx"
}
```

Although it may well be a generator bug that we don't support this yet even though Node.js does!

Reply:
How to fallback to `.ts` if `.tsx` not found?

Reply:
You need to know in the mapping itself which extension to use - exports are designed to be usable without having to check the filesystem.